src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days > 6) {
+    return (days * 40) - 50;
+  };
+
+  if (days > 2) {
+    return (days * 40) - 20;
```

Comment:
Also you should add a variable with the rent cost per day (40), because you use it multiple times and it will be easier to change it if needed.